app/controllers/shipit/shipit_controller.rb

```diff
@@ -10,12 +10,14 @@ class ShipitController < ApplicationController
 
     helper Shipit::Engine.routes.url_helpers
     include Shipit::Engine.routes.url_helpers
+    include ActionView::Helpers::DateHelper
```

Comment:
Because if that helper have dependency in another helper you will have to track it and also include here and sometimes it is really hard to track all dependencies of a helper.